pkg/endpoint/restore_test.go

```diff
@@ -84,6 +84,11 @@ func TestReadEPsFromDirNames(t *testing.T) {
 
 	const unsupportedTestOption = "unsupported-test-only-option-xyz"
 
+	cwd, err := os.Getwd()
```

Comment:
(cannot comment on the actual line, thus adding the comment here)

Not directly related, but since there's a linter error about this anyway: this test could be switched to use [`testing.T.TempDir`](https://pkg.go.dev/testing#F.TempDir) instead of the manual `os.MkdirTemp` and the deferred `os.RemoveAll`.

Reply:
Changed to use `t.TempDir()`, thanks for the hint!